src/index.coffee

```diff
@@ -46,12 +46,17 @@ class Room extends Hubot.Adapter
     @privateMessages[envelope.user.name].push ['hubot', str] for str in strings
 
 class Helper
+  @Response = MockResponse
```

Comment:
The inclusion of this just allows for more simple mocking from the test cases.

```coffeescript
class NewMockResponse extends Helper.Response
```

rather than
```coffeescript
Hubot = require('hubot')
class NewMockResponse extends Hubot.Response
```